components/brave_wallet/common/common_util.cc

```diff
@@ -29,4 +29,18 @@ bool IsAllowed(PrefService* prefs) {
   return !IsDisabledByPolicy(prefs);
 }
 
+std::vector<uint8_t> Leb128Encode(size_t value) {
```

Comment:
Needs comment with the spec link.
Also needs unit tests based on some public test vectors.

Reply:
Removed